solidity/ExpirableAirdrop.sol

```diff
@@ -1,60 +1,106 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity >=0.8.0;
 
-import '@openzeppelin/contracts/access/Ownable.sol';
+import './utils/Governable.sol';
 import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
 import '@openzeppelin/contracts/utils/cryptography/MerkleProof.sol';
 
 /// @title ExpirableAirdrop
 /// @notice airdrop claimable by members of a merkle tree with expiration date
 /// @author Mean Finance (based on: https://github.com/Anish-Agnihotri/merkle-airdrop-starter/blob/master/contracts/src/MerkleClaimERC20.sol)
-contract ExpirableAirdrop is Ownable {
+contract ExpirableAirdrop is Governable {
   IERC20 public immutable token;
   uint256 public immutable expirationTimestamp;
   bytes32 public immutable merkleRoot;
   mapping(address => bool) public hasClaimed;
 
   error AlreadyClaimed();
   error Expired();
+  error NotExpired();
   error NotInMerkle();
 
-  event Claimed(address indexed to, uint256 amount);
+  event Claimed(address indexed claimee, address indexed receiver, uint256 amount);
+  event Deposited(uint256 amount);
+  event Retrieved(uint256 timestamp, address receiver);
 
   /// @notice Creates a new ExpirableAirdrop contract
   /// @param _token token to airdrop
+  /// @param _expirationTimestamp expiration timestamp
   /// @param _merkleRoot of claimees
   constructor(
     IERC20 _token,
     uint256 _expirationTimestamp,
     bytes32 _merkleRoot
```

Comment:
Let's do this in a different pull request, but I think this should be configurable (updatable). Based on [0xngmi's reflections](https://github.com/tubby-cats/tubby-launch-reflections), we might need to update it if we make a mistake when building the tree